main.lua

```diff
@@ -1,4 +1,31 @@
 -- check if the api definitions are present in the lib
+header = [[
+# Lutro Status
+
+Compares the [Lutro](https://github.com/libretro/libretro-lutro) API to the [LÖVE API](https://github.com/love2d-community/love-api), resulting in a [`lutro-status`](lutro-status.txt) file.
+
+## Requirements
+
+- [`git`](https://git-scm.com/)
+- [`make`](https://www.gnu.org/software/make/)
+- [RetroArch](http://www.libretro.com)
+- [libretro-lutro](https://github.com/libretro/libretro-lutro)
+
+## Usage
+
+```
+make
+```
+
+### Build Manually
+
+```
+git submodule update --init
+retroarch -L /usr/lib/libretro/lutro_libretro.so . > lutro-status.txt
+```
+
+## Status
+]]
```

Comment:
This is redundant with the README.template.md. It should be possible to read the template from lua/lutro/löve.
If you prefer to have the contents of the template file in the script, the template file should be removed.